### ISSUE TITLE
Set an Environment in a Pre-Request Script

### ISSUE THREAD

**Issue by mikelietz** [a85/POSTMan-Chrome-Extension#9208]
**Describe the solution you'd like**
I have over 20 environments, often three for various tiers of the same application. I can set the tier in variables like {{{{env}}_url}} but I still need to select the proper environment every time. When I'm comparing production and development versions, this leads to a lot of extra clicking.

I'd like to be able to do something like
`postman.setEnvironment("520 DEV");`

**Comment by MarcelloModica** [comment]
Did you find a solution?

**Comment by mikelietz** [comment]
No, I was never able to find a programmatic way to set the environment. I thought there was an answer given somewhere to the effect of "There is currently no way to do this." but I can't find it.

**Comment by wlcharlie** [comment]
need this!
